src/components/draft/editor/FileSelector.tsx

```diff
@@ -0,0 +1,39 @@
+import { List, ListItemButton, ListItemText } from '@mui/material';
+import useEditorStore, {
+    editorStoreSelectors,
+} from 'components/draft/editor/Store';
+import useDraftSpecs from 'hooks/useDraftSpecs';
+import { FormattedMessage } from 'react-intl';
+
+function EditorFileSelector() {
+    const setCurrentCatalog = useEditorStore(
+        editorStoreSelectors.setCurrentCatalog
+    );
+    const currentCatalog = useEditorStore(editorStoreSelectors.currentCatalog);
+    const draftId = useEditorStore(editorStoreSelectors.draftId);
+    const { draftSpecs } = useDraftSpecs(draftId);
+
+    return (
```

Comment:
The conditionals in this return statement are a little challenging to follow. Could they be written as seen in the following example?

```ts
return draftId && (
  <List>
    { draftSpecs.length > 0 ? (<ComponentSeriesOne />) : (<ComponentSeriesTwo />) }
  </List>
);
```

Reply:
Nope - it is not the same thing. This is a weird side effect of out styling from linting and how it handled the null response. Since it is such a short line it will always move itself to a single line. So when I had it near the bottom it looked even more confusing.